pre_cache_action.sh

```diff
@@ -32,6 +32,19 @@ input_packages="${@:6}"
 
 # Trim commas, excess spaces, and sort.
 log "Normalizing package list..."
+
```

Comment:
Move this block to a library function in lib.sh and have it used in both locations